spec/attribute.dd

```diff
@@ -86,6 +87,10 @@ $(GNAME LinkageType):
     $(D Windows)
     $(D System)
     $(D Objective-C)
+
+$(GNAME NamespaceList):
+    $(GLINK2 expression, ConditionalExpression)
+    $(GLINK2 expression, ConditionalExpression)$(COMMA) $(I NamespaceList)
```

Comment:
This allows trailing comma's:
```suggestion
    $(GLINK2 expression, ConditionalExpression)
    $(GLINK2 expression, ConditionalExpression)$(COMMA)
    $(GLINK2 expression, ConditionalExpression)$(COMMA) $(I NamespaceList)
```

(Don't think i can write on PR's here)

Reply:
When did this change? I get an error when I leave a trailing comma.

Reply:
Fixed that one recently in dlang/dmd#11090